src/uniprotkb/components/statistics/HistoricalReleasesEntriesLinePlot.tsx

```diff
@@ -0,0 +1,113 @@
+import { memo, useCallback, useEffect, useRef } from 'react';
+import {
+  select,
+  scaleLinear,
+  axisBottom,
+  axisLeft,
+  line,
+  scaleTime,
+  format,
+  timeFormat,
+} from 'd3';
+
+import styles from './styles/historical-release-entries-line-plot.module.scss';
+
+// Specify the chart’s dimensions.
+const width = 400;
+const height = 400;
+const margin = { top: 20, right: 60, bottom: 45, left: 80 };
+
+export type DateCount = [Date, number];
+export type Bounds = {
+  date: [Date, Date];
+  count: [number, number];
+};
+
+type Props = {
+  dateCounts?: DateCount[];
+  bounds?: Bounds;
+};
+
+const HistoricalReleasesEntriesLinePlot = ({ dateCounts, bounds }: Props) => {
+  const svgRef = useRef<SVGSVGElement>(null);
+
+  const renderPlot = useCallback((dateCounts: DateCount[], bounds: Bounds) => {
+    const chart = select(svgRef.current).select('g');
+
+    // x-axis
+    const xScale = scaleTime()
+      .domain(bounds.date) // units: Date
+      .range([0, width]); // units: pixels
+    chart
+      .select<SVGGElement>('.x-axis')
+      .transition()
+      .duration(1_000)
+      .call(
+        axisBottom(xScale).tickFormat((d: Date | { valueOf(): number }) =>
+          timeFormat('%Y')(new Date(d.valueOf()))
+        )
```

Comment:
```suggestion
        axisBottom(xScale)
          .ticks(timeYear)
          .tickFormat((d: Date | { valueOf(): number }) =>
            timeFormat('%Y')(new Date(d.valueOf()))
          )
```
Here, make sure that we get only 1 tick every year on January 1st

Reply:
Also, maybe with the full data we'll have to adjust the ticks even more and do `timeYear.every(2)` for example

Reply:
![image](https://github.com/user-attachments/assets/0d46c218-28bb-417a-b4d3-cd58a8ca7b71)

Current plot has every 4 years - shall we just go for that?